covid_act_now/params.json.template

```diff
@@ -13,5 +13,33 @@
       "aws_access_key_id": "",
       "aws_secret_access_key": ""
     }
+  },
+  "validation": {
+    "common": {
+      "data_source": "covid-act-now",
+      "span_length": 14,
+      "end_date": "today-5",
+      "min_expected_lag": {"all": "3"},
+      "max_expected_lag": {"all": "6"},
+      "suppressed_errors": [
+        {"check_name": "check_se_many_missing",
+         "signal": "pcr_specimen_total_tests"},
+        {"check_name": "check_se_not_missing_and_in_range",
+         "signal": "pcr_specimen_total_tests"},
+        {"check_name": "check_n_missing",
+         "signal": "pcr_specimen_total_tests"},
+        {"check_name": "check_se_0_when_val_0"}         
+      ]
+    },
+    "static": {
+      "minimum_sample_size": 100,
```

Comment:
Covid Act Now was the one with really low sample sizes (like 0.7) being produced, right? Were you able to verify that we actually want to report those, or is 100 the desired sample size threshold?

Reply:
Yep, I'll need to change that - Katie says treat it as 0 for now